.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
.vscode settings file should not be pushed to the public repo. I'm sorry this hasn't already been added to `.gitignore`

Reply:
should I add /.vscode to .gitignore ?